src/app/components/footer/footer.component.html

```diff
@@ -1,17 +1,19 @@
-<div id="footer-container">
-  <div id="footer-links-container">
-    <a href="https://discord.gg/pwsTq4Yt9q" target="_blank"><p>Discord</p></a>
-    <a href="https://twitter.com/impact_moe" target="_blank"><p>Twitter</p></a>
-    <a href="https://github.com/impact-moe" target="_blank"><p>Github</p></a>
-  </div>
-  <div id="footer-credits-container">
-    <p>
-      Impact website, software and tools are open source and licensed under the
-      GNU General Public License v3.0.
-    </p>
-    <p>
-      Data collected from the Genshin Wiki, Honey Hunter World, and the Genshin
-      Impact Data Gathering channel.
-    </p>
+<div class="footer-container">
+  <div class="footer-content">
+    <div class="footer-links-container">
+      <a href="https://discord.gg/pwsTq4Yt9q" target="_blank">Discord</a>
```

Comment:
Should probably add `rel="noopener"` or `rel="noreferrer"` to these links for better performance & security.

https://web.dev/external-anchors-use-rel-noopener/